plugins/bundle/plugin.go

```diff
@@ -310,14 +312,19 @@ func (p *Plugin) UnregisterBulkListener(name interface{}) {
 
 // Config returns the plugins current configuration
 func (p *Plugin) Config() *Config {
-	return &p.config
+	p.cfgMtx.RLock()
+	defer p.cfgMtx.RUnlock()
+	return &Config{
+		Bundles: p.getBundlesCpy(),
```

Comment:
Isn't at least the `Name` field also still used through `IsMultiBundle()`? https://github.com/open-policy-agent/opa/blob/08d295f5201ac8c9702c48b06626967cb25edd36/plugins/discovery/discovery.go#L736

Reply:
Yeah working on a fix.

Reply:
Tests are green now.